runtime/process.go

```diff
@@ -201,7 +201,19 @@ func (p *process) Stdio() Stdio {
 
 // Close closes any open files and/or resouces on the process
 func (p *process) Close() error {
-	return p.exitPipe.Close()
+	err := p.exitPipe.Close()
+
+	if err != nil {
+		return err
```

Comment:
Just log a warning here or in the unlikely case that `Close` errors out, the `controlPipe` will never be closed.